src/main/java/org/folio/mr/controller/MediatedRequestLoansController.java

```diff
@@ -18,12 +22,21 @@
 public class MediatedRequestLoansController implements MediatedRequestsLoansApi {
 
   private final CheckOutService checkOutService;
+  private final DeclareLostService declareLostService;
 
   @Override
   public ResponseEntity<CheckOutResponse> checkOutByBarcode(CheckOutRequest request) {
     return ResponseEntity.ok(checkOutService.checkOut(request));
   }
 
+  @Override
+  public ResponseEntity<Void> declareItemLost(UUID loanId,
+    DeclareLostCirculationRequest declareLostRequest) {
+
+    declareLostService.declareLost(loanId, declareLostRequest);
+    return ResponseEntity.noContent().build();
```

Comment:
I'm not sure if returning a `ResponseEntity` from a service is a good idea. That's controller's responsibility, isn't it?

Reply:
That's a controller, DeclareLostService service doesn't return anything